### PR TITLE
Feature/sentiment numeric

### PR DESCRIPTION
**Pull request by hollberg**
## Description

This PR adds a 'sentiment_numeric' endpoint that expands upon the existing 'sentiment' endpoint. The existing endpoint returns "human-readable" sentiment scores of "Positive", "Negative", and "Neutral", which are great for summarizing and high level classification of sentiments. By adding a sentiment analyzer that returns numeric sentiment values (within ranges of -1 to +1), we enable more granular analysis of sentiment, opening the door for evaluating things such as trend analysis, forecasting, and more nuanced investigations into sentiment.

Authors:
@Zack-Quintana 
@BrandonSmith710 
@hollberg  

Fixes # (issue)
https://trello.com/c/h5AS5v84/285-as-a-data-science-developer-i-want-the-option-to-classify-texts-sentiment-in-either-human-terms-ie-positive-or-negative-or-with

## Type of change

- [x] New feature

## Checklist:

- [x] My code follows PEP8 style guide
- [x] I have removed unnecessary print statements from my code
- [x] I have made corresponding changes to the documentation if necessary
- [x] My changes generate no errors
- [x] No commented-out code
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

Loom Video:
https://www.loom.com/share/bbac425c588040db8a62734a6336c13d

### CODE REVIEW

**Comment by BrokenShell** [comment]
We intentionally converted the numeric output to english, we really don't need both, right?

**Comment by hollberg** [comment]
> We intentionally converted the numeric output to english, we really don't need both, right?

I think we do need both, depending on the use case. Converting to English makes for a nice human-readable response, but throws away lots of valuable information. Having a numeric score allows us to view things like trends (are meeting negative but getting less negative over time), and will be useful for future predictive regression (vs classification) models (ie, predict average meeting sentiment base on mentor/mentee attributes).

**Comment by hollberg** [comment]
ok so what do I do now?

**Comment by BrokenShell** [comment]
Let's make a graph of the feedback in our database and then create an endpoint that delivers the graph as json.

File: app/api.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 from app.data import MongoDB
 from app.utilities import financial_aid_gen
 from app.model import MatcherSortSearch, MatcherSortSearchResource
-from app.vader_sentiment import vader_score
+from app.vader_sentiment import vader_score, vader_numeric
```
Comment (BakerJr1904) [inline app/api.py:10]: I've never seen Vader_numeric before thank you!!

**Review by BakerJr1904 — COMMENTED** [review]
This is awesome, I was happy to see new stuff.

**Pull request closed without merging**